Dockerfile

```diff
@@ -36,7 +36,8 @@ RUN curl -fsSL https://mafft.cbrc.jp/alignment/software/mafft-7.475-linux.tgz \
 WORKDIR /build/RAxML
 RUN curl -fsSL https://api.github.com/repos/stamatak/standard-RAxML/tarball/v8.2.12 \
   | tar xzvpf - --no-same-owner --strip-components=1
-RUN make -f Makefile.AVX.PTHREADS.gcc   # AVX should be widely-supported enough
+# AVX should be widely-supported enough
+RUN if [ "$TARGETPLATFORM" = "linux/amd64" ]; then make -f Makefile.AVX.PTHREADS.gcc; fi
```

Comment:
Oof. I guess it really is [as painfully slow as we were warned](https://github.com/nextstrain/docker-base/issues/35#issuecomment-1204318267).

If we're going to make this not slow by building arm64 without emulation, then we'd need to provision our own runner on our "own" hardware (e.g. rented from AWS or even a purchased M1/M2 Mac Mini) and use it in CI here. That has implications for [how the final multi-arch image is produced](https://github.com/nextstrain/docker-base/pull/50#discussion_r955533198). Since we'd no longer have the option of doing it all in one go, we'd have to upload the intermediate OCI tarballs as job artifacts from each native host (e.g. x86_64 and arm64) and then combine them afterwards with `docker manifest` in another job (e.g. like Nextstrain CLI arranges for its standalone builds).

Reply:
Ah, one further note about this idea of a Mac Mini as a build machine (mentioned above and [earlier in Slack](https://bedfordlab.slack.com/archives/C01LCTT7JNN/p1654885748224469?thread_ts=1654883833.977699&cid=C01LCTT7JNN)): currently the GitHub Actions runner [doesn't support arm64 on macOS](https://docs.github.com/en/actions/hosting-your-own-runners/about-self-hosted-runners#architectures), so that's moot for now (though I suspect it's a popularly requested feature).

Reply:
And it may be possible to enable dev/experimental support for arm64 on macOS, c.f. https://www.pantsbuild.org/docs/ci-for-macos-on-arm64#setting-up-the-self-hosted-runner.

Reply:
Another note in that vein: for our arm64 image builds here, we don't actually need macOS on arm64 hardware as we'd be running Linux in a VM on it in that case anyway. So Linux on arm64 hardware is actually preferable and these are available for rent from many sources as previously noted.

But for _testing_ our arm64 images on macOS in CI, we would need macOS running on arm64 hardware. We would also need that for building macOS arm64 standalone binaries for the CLI too.

Reply:
> though I suspect it's a popularly requested feature

Looks about right: https://github.com/actions/runner-images/issues/2187

Reply:
Back to the main issue here (`arm64` emulation slowness).

> Linux on arm64 hardware is actually preferable and these are available for rent from many sources as previously noted.

This looks like a good guide detailing how to deploy/use arm64 self-hosted runners on AWS: https://noise.getoto.net/2021/11/01/building-arm64-applications-on-aws-graviton2-using-the-aws-cdk-and-self-hosted-runners-for-github-actions/

Reply:
> However, it's probably reasonable to use emulation for a bit and [stay hopeful](https://twitter.com/iximiuz/status/1565629586880270336) for GitHub-hosted arm64 runners.

Agreed.

> This looks like a good guide detailing how to deploy/use arm64 self-hosted runners on AWS: https://noise.getoto.net/2021/11/01/building-arm64-applications-on-aws-graviton2-using-the-aws-cdk-and-self-hosted-runners-for-github-actions/

Nod. Like most how-to guides of this style, I would suggest using it as a rough list of steps to consider but not to actually copy/paste the examples nor treat it as a comprehensive (or even good) way to set the thing in question up. An artifact of a specific person's choices best used to guide your own understanding and support your own learning of how to set up the thing for your needs with your own choices.

Reply:
I wrote down [some of my thoughts on self-hosted runners](https://github.com/tsibley/blab-standup/blob/master/2022-09-14.md#readme).

Reply:
A better solution for speeding up build time is **cross-compilation** mentioned in [the buildkit docs](https://github.com/moby/buildkit/blob/f2c41e15cb4d725649efe47a8d3023d0b35fdbe8/docs/multi-platform.md#builds-are-very-slow-through-emulation) and in [this guide](https://medium.com/@tonistiigi/faster-multi-platform-builds-dockerfile-cross-compilation-guide-part-1-ec087c719eaf). Basically, use `FROM --platform=$BUILDPLATFORM` to avoid emulation. I'll give it a shot in this PR.

Reply:
Ah, nice. We'd still need emulation (or self-hosted runners) for testing though, yeah?

(And I think also need them for other contexts/purposes where we can't cross-compile, but unrelated to this image.)

Reply:
Yes, definitely!

Reply:
Moving cross-compilation discussion to https://github.com/nextstrain/docker-base/pull/50#discussion_r986331019.